src/contrasts.jl

```diff
@@ -233,7 +233,7 @@ levels needs to be converted into ``k`` model matrix columns instead of the
 standard ``k-1``.  This occurs when there are missing lower-order terms, as in
 discussed below in [Categorical variables in Formulas](@ref).
 
-**Examples**
+# Examples
 
 ```julia
```

Comment:
Sorry, I didn't intend to comment on that line. Examples should indeed be blocks (and even better `jldoctest` blocks). I wanted to refer to the constructor signatures on a different line.